tests/Api/DocData/TestCase.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+
+namespace HomeCEU\Tests\Api\DocData;
+
+
+class TestCase extends \HomeCEU\Tests\Api\TestCase {
+
```

Comment:
just in case you need it later?

Reply:
lol, fairly sure I had something in here, then perhaps I moved it out, hard to say.